cmd/snap-exec/main_test.go

```diff
@@ -242,3 +242,52 @@ foobar
 
 `)
 }
+
+func (s *snapExecSuite) TestSnapExecHookRealIntegration(c *C) {
+	// we need a lot of mocks
+	dirs.SetRootDir(c.MkDir())
```

Comment:
Yeah, the [teardown](https://github.com/snapcore/snapd/blob/master/cmd/snap-exec/main_test.go#L54).